### DISCUSSION TITLE
Ability to create a shared dotfile of custom commands for collaboration

### DISCUSSION THREAD

**Discussion by rahulrai-in** [Azure/vscode-aks-tools#428] [Ideas]
**Is your feature request related to a problem? Please describe.**
Currently, the tool is opinionated about the commands that developers want to execute. As a developer, I would like to maintain a library of commands that I can easily execute from the extension. Also, the default commands should be configurable. For example, I prefer to include the "-o wide" parameter when using the `kubectl get` commands.

**Mention what platform you want to support the new feature**
VSCode

**Describe the solution you'd like**
A dotfile, preferably named `.aks` that I can commit to the code repo with a dictionary of custom commands and options to modify the behavior of existing ones.

**Describe alternatives you've considered**
This is a common method used by many tools to customize their behavior and hence I did not consider other alternatives.

**Additional context**
An advantage of using dotfiles is that they can be easily distributed to the wider community through popular repositories: https://github.com/webpro/awesome-dotfiles

**Comment by rahulrai-in** [comment]
Tagging @nellysattari for additional recommendations.

**Comment by Tatsinnit** [comment]
Hiya, I am moving this to the discussion forum now, given this is now U/X'd well with new design (#226 ), which is well improved, lets get the feelers for how many use-cases will need specific online file support.

#226 - was the drop of the pull request which did re-design for the this `kubectl` commands for helpful developer experience. https://azure.github.io/vscode-aks-tools/features/run-kubectl-command.html 

Thank you so much.
